cfgov/v1/util/util.py

```diff
@@ -143,3 +144,32 @@ def all_valid_destinations_for_request(request):
                             valid_destination_for_request(request, pair[1])]
 
     return valid_destinations
+
+
+def load_class(full_class_string):
```

Comment:
See above; this is useful, but a duplicate of existing Django functionality in [`django.utils.module_loading.import_string`](https://docs.djangoproject.com/en/1.8/ref/utils/#django.utils.module_loading.import_string).